packages/flutter/lib/src/rendering/debug.dart

```diff
@@ -89,3 +104,32 @@ List<String> debugDescribeTransform(Matrix4 transform) {
   matrix.removeLast();
   return matrix;
 }
+
+/// Returns true if none of the rendering debug variables have been changed.
+///
+/// This function is used by the test framework to ensure that debug variables
+/// haven't been inadvertently changed.
+bool debugAllRenderVarsUnset() {
+  return !(
```

Comment:
ideally these debug*Unset functions would have their bodies entirely in an assert() (maybe except for a result variable declared before and returned after). That way the debug variables aren't ever accessed outside asserts, which will be easier for the compiler to prove.